bin/build-hdfs

```diff
@@ -9,6 +9,7 @@ IMPALA_ZIP=$IMPALA_VER.zip
 IMPALA_URL=https://github.com/cloudera/Impala/archive/$IMPALA_ZIP
 NATIVE=native
 DIST=hdfs-mesos-0.1.0
+DOCKER_IMAGE_NAME=hdfs-mesos
```

Comment:
It's the same variable as below. It's overridden if an image name is specified as $2